lib/install/management/virtual_app.go

```diff
@@ -30,47 +30,34 @@ import (
 func (d *Dispatcher) createVApp(conf *config.VirtualContainerHostConfigSpec, settings *data.InstallerData) (*object.VirtualApp, error) {
 	defer trace.End(trace.Begin(""))
 	var err error
-
+	var zero int64
 	log.Infof("Creating virtual app %q", conf.Name)
 
-	resSpec := types.ResourceConfigSpec{
-		CpuAllocation: &types.ResourceAllocationInfo{
-			Shares: &types.SharesInfo{
-				Level: types.SharesLevelNormal,
-			},
-			ExpandableReservation: types.NewBool(true),
-		},
-		MemoryAllocation: &types.ResourceAllocationInfo{
-			Shares: &types.SharesInfo{
-				Level: types.SharesLevelNormal,
-			},
-			ExpandableReservation: types.NewBool(true),
-		},
-	}
+	resSpec := types.DefaultResourceConfigSpec()
 	cpu := resSpec.CpuAllocation.GetResourceAllocationInfo()
-	cpu.Limit = -1
-	if settings.VCHSize.CPU.Limit != 0 {
+
+	if settings.VCHSize.CPU.Limit != nil && *settings.VCHSize.CPU.Limit != zero {
```

Comment:
I'm fine with just using `0` - I don't see `&zero` used anyway so I'm guessing this is a hold over from working changes?

Reply:
yeah...I think this was still when the fog of vic-machine was thick....